src/test/moment/start_end_of.js

```diff
@@ -386,6 +386,20 @@ test('startOf across DST -1', function (assert) {
     moment.updateOffset = oldUpdateOffset;
 });
 
+test('endOf DST day', function (assert) {
+    // Based on a real story somewhere in Brazil/Sao_Paulo
+    var dstChangeDay = moment('2017-10-15T12:00:00-03:00');
+    var dstChangeDayAfter = moment('2017-10-16T12:00:00-03:00');
```

Comment:
Assuming you mean `America/Sao_Paulo`, the offset would be `-02:00` the day after.

The problem here is that this test will be running on whatever the local time zone happens to be - not necessarily in Brazil.

I think though the code change needs to happen in moment, we would need to bring moment-timezone into the unit tests (or move the tests into moment-timezone) in order to start testing DST changes.

Reply:
I copied another test in same file to make my, both should be moved?

Reply:
Do I remove the test and left just the code?

Reply:
@mj1856

Reply:
@mj1856 , removed tests. I'll try to make them on https://github.com/moment/moment-timezone